steering-files/src/main/resources/org/hps/steering/recon/PhysicsRun2019_pass0_recon.lcsim

```diff
@@ -337,7 +337,7 @@
             <maxMatchDt>40</maxMatchDt>
             <trackClusterTimeOffset>40</trackClusterTimeOffset>
             <useCorrectedClusterPositionsForMatching>false</useCorrectedClusterPositionsForMatching>
-            <applyClusterCorrections>true</applyClusterCorrections>
+            <applyClusterCorrections>false</applyClusterCorrections>
```

Comment:
The correction needs to be applied only once. In the PhysicsRun2019_pass0_KFOnly.lcsim steering file we only construct ReconstructedParticles once, so the corrections need to be applied. In the PhysicsRun2019_pass0_recon.lcsim steering file, we create ReconstructedParticles twice, so the corrections should be applied only once, and since the KF ReconstructedParticles are being created after the GBL ReconstructedParticles, the corrections are turned off the second time through.

Reply:
Ok. I see